Src/FluentAssertions/Common/Guard.cs

```diff
@@ -8,10 +8,7 @@ internal static class Guard
     {
         public static void ThrowIfArgumentIsNull<T>([ValidatedNotNull] T obj, string paramName)
         {
-            if (obj is null)
-            {
-                throw new ArgumentNullException(paramName);
-            }
+            ThrowIfArgumentIsNull(obj, paramName, string.Empty);
```

Comment:
Are you aware that this changes behavior?
I think the former, i.e. keeping it as it is, is better.
![image](https://user-images.githubusercontent.com/919634/161090048-f00aafeb-50af-4e7b-bcdf-f4ee3bacd854.png)

Also applies to the change to `ThrowIfArgumentIsNullOrEmpty`.